packages/manager/modules/enterprise-cloud-database/src/service/details/details.html

```diff
@@ -0,0 +1,29 @@
+<oui-page-header data-heading="{{ $ctrl.clusterDetails.name }}" data-description="{{ $ctrl.clusterDetails.id }}">
+    <oui-header-tabs>
+        <oui-header-tabs-item
+            data-text="{{ ::'enterprise_cloud_database_service_details_overview' | translate }}"
+            data-state="enterprise-cloud-database.service.details.overview"></oui-header-tabs-item>
```

Comment:
states should go in the routing's resolve and attribute `href` should be used instead to get routing abstraction in the views

Reply:
oui-header-tabs-item currently does not support a function in href. Even if we use on-click attribute to call a function (which is in resolve), we will have to set href to '/#', which results in wrong behaviour. Please let me know if you have a different way to do it